source/_integrations/cambridge_audio.markdown

```diff
@@ -68,6 +68,13 @@ The integration provides a few entities to configure the device settings. The fo
 - Pre-Amp
 - Early update
 - Audio output (Speaker select)
+- Control Bus mode
+
+### Pre-Amp
+When Pre-Amp mode is enabled, Home Home Assistant can control the output volume of your Pre-Amplifier. 
+
+### Control Bus
+When Control Bus mode is ebabled, Home Assistant can control the output volume of your Power Amplifier when it is connected to a Cambridge Audio network player using the Control Bus interface. In this case Pre-Amp mode can be disabled, the metwork player will send the signal with full volume to the power amplifier and the volume of the power amplifier can be controlled using volume up and down controls. Control Bus does not support setting the volume to a certain value, it can only increase and decrease the current volume.
```

Comment:
```suggestion
When Control Bus mode is enabled, Home Assistant can control the output volume of your Power Amplifier when it is connected to a Cambridge Audio network player using the Control Bus interface. In this case Pre-Amp mode can be disabled, the network player will send the signal with full volume to the power amplifier and the volume of the power amplifier can be controlled using volume up and down controls. Control Bus does not support setting the volume to a certain value, it can only increase and decrease the current volume.
```